deploy/config-spec/properties.yaml

```diff
@@ -137,7 +137,7 @@ properties:
           value: "8443"
       roles:
         - name: "coordinator"
-          required: false
+          required: true
```

Comment:
The HTTPS port is required now; I thought we were making a move to not have ports configurable anymore?

Reply:
Its not configurable.